elixir/apps/web/lib/web/live/settings/identity_providers/microsoft_entra/components.ex

```diff
@@ -96,6 +96,16 @@ defmodule Web.Settings.IdentityProviders.MicrosoftEntra.Components do
                   </p>
                 </div>
               </.inputs_for>
+
+              <p class="text-sm text-neutral-500">
+                <strong>Note:</strong>
```

Comment:
Should we explain every limit on the billing page instead of putting it everywhere?

Reply:
Based on customer calls, this is the point where they are wondering if the synced users count against their billing limit, so I wanted to clear it up here. I updated the subscription metadata for each customer so that the limit is explained better on the billing page as well.